packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/request.ts

```diff
@@ -14,7 +14,7 @@ export function requestScopesFactory(
   return async function requestScopes(scopes: string[]) {
     const {logger} = params;
 
-    logger.debug('Request optional scopes: ', {
+    logger.debug('Requesting optional scopes: ', {
       shop: session.shop,
       ...{scopes},
     });
```

Comment:
Nit: can we simplify the context here?
```suggestion
    logger.debug('Requesting optional scopes: ', {shop: session.shop, scopes});
```